src/crimson/osd/ops_executer.h

```diff
@@ -84,6 +84,7 @@ class OpsExecuter {
   PG& pg;
   PGBackend& backend;
   Ref<MOSDOp> msg;
+  osd_op_params_t osd_op_params;
```

Comment:
How costly is the construction? Maybe `std::optional<>` could help. I'm a little bit afraid about the hot path here.